### PR TITLE
O11Y-2238 moved to api

### LINKED ISSUE #80: Global setup code should be exposed via the api not via the sdk
**Describe the bug**
Currently, the code responsible for setting up Global objects is accessible from the sdk.  This is contrary to the OTEL convention for libraries to only depend on the API.

We should expose the Global objects code from the API instead.

### PR DESCRIPTION
**Pull request by danrick-wk**
## Which problem is this PR solving?

Global setup code should be exposed via the api not via the sdk
Fixes #80 

## Short description of the changes

 - moved the opentelemetry.dart file into the API directory 
 - updated api.dart & sdk.dart to support the file move
 - updated testing files to support the file move

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

@dustinlessard-wf will test this with a Flutter App

## Checklist:

- [x] Followed the style guidelines of this project

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by semveraudit-wf** [comment]
### Public API Changes

 No changes to the public API found for commit 9c5040107ec9bfd958035a11b0fb5b25276ef94a

Showing results for 9c5040107ec9bfd958035a11b0fb5b25276ef94a
> Powered by [semver-audit-service](https://github.com/Workiva/semver-audit-service). Please report any problems by [filing an issue](https://github.com/Workiva/semver-audit-service/issues).
> Reported by the dart semver audit client 2.2.2
[Browse public API](https://semver-audit.wk-dev.wdesk.org/Workiva/opentelemetry-dart/9c5040107ec9bfd958035a11b0fb5b25276ef94a).

**Review by blakeroberts-wk — DISMISSED** [review]

**Review by dustinlessard-wf — APPROVED** [review]

**Comment by dustinlessard-wf** [comment]
Let's defer documentation updates until we have completed the evaluation of adopting the OTEL Java pattern of having a top level global OpenTelemetry singleton.

**Comment by dustinlessard-wf** [comment]
QA +1 , verified api noop functionality using a Flutter test app
Screen Shot 2022-09-09 at 4 20 11 PM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.15.0
+
+- Setup code exposed through API (no longer exposed through SDK)
```
Comment (evanweible-wf) [inline CHANGELOG.md:3]: #nit since this package is open source now, it might be helpful to list the APIs that have been moved here.

**Comment by dustinlessard-wf** [comment]
@evanweible-wf @blakeroberts-wk , Please don't re-review at this time.  I have some comments to address first. THANKS!

File: CHANGELOG.md (hunk 1 above)
Comment (danrick-wk) [reply CHANGELOG.md]: Updated

**Review by evanweible-wf — APPROVED** [review]

**Review by dustinlessard-wf — APPROVED** [review]

**Comment by dustinlessard-wf** [comment]
QA +1 , verified api noop functionality using a Flutter test app

**Comment by dustinlessard-wf** [comment]
@Workiva/release-management-p

**Comment by dustinlessard-wf** [comment]
semver +1 @Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**